typegate/engine/Cargo.toml

```diff
@@ -34,19 +34,21 @@ mt_deno.workspace = true
 common.workspace = true
 typescript.workspace = true
 
-# remove wasmedge once python runtime has been migrated
-wasmedge-sdk.workspace = true
-wasmedge-sys.workspace = true
-wasmedge-types.workspace = true
-
-wasmtime = { version = "19.0.1",  features = ["component-model"] }
+wasmtime = { workspace = true,  features = ["component-model"] }
+wasmtime-wasi.workspace = true
+# remove when bumping wasmtime to 20
+cap-std = "3"
```

Comment:
After some investigations, there shouldn't probably be major changes when bumping to 20 on the type conversion stuff in case some of this depends on that. They pretty much changed how to instantiate values in a more 'non-confusing' way.